ping_examples/core-ping.json

```diff
@@ -0,0 +1,34 @@
+{
+  "metadata": {
+    "ping_type_name": "core",
+    "client_id": "f99fbfb6...",
+    "app_build": "59f330e5...",
+    "telemetry_sdk_build": "abcdabcd...",
+    "seq": 1
+  },
+  "metrics": {
+    "string": {
+      "device.os": "android",
+      "device.os_version": "8.0.0",
+      "device.device": "Motorola X4",
+      "device.architecture": "arm"
+    },
+    "number": {
+      "device.tz": -5
+    },
+    "string_list": {
+      "device.accessibility_services": [
+        "screen_reader",
+        "contrast"
+      ]
+    }
+  },
+  "errors": {
```

Comment:
On a second thought, what do we gain by having the error metrics in their own section? We could keep generic and just consider them a normal metric, reported in their related section

Reply:
So they would all be under a `labeled_counts` section?  Or do you mean with each metric, as in:

```
"metrics": {
  "number": {
    "open_tabs": {
       "value": 4,
       "errors": {
           "invalid_value": 2
      }
   }
}
```
?

Reply:
I mean labeled_counts: this way we won't need any specialized code to persist errors, we could simply treat that as any other metric. The only difference, is that it would be a core telemetry health metric :)

Reply:
I'm a big fan of this idea. It would make things simpler on our end as well, when we transform the data and make it queryable.

Reply:
Ok.  Seems like a win all around.  I'll update this.